easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.2.4'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
```

Comment:
Can't seem to download the sources right now... `An error occurred when sending the email. `

Reply:
True, I get the same thing. Eventually you end up there: ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest
and it seems they might be moving to Github, but I didn't want to rely on this: https://github.com/alekseyzimin/masurca/releases (as the Github account seems to be a personal one, the original website is institutional). 
Should I change it for the direct FTP link?

Reply:
I'd say yes, if we can auto-download, we should?